pkg/ako/ako_test.go

```diff
@@ -64,13 +64,13 @@ var _ = Describe("AKO", func() {
 		})
 	})
 
-	When("no status is in StatefulSet", func() {
+	When("no annotation is in StatefulSet", func() {
 		It("should not claim finished", func() {
 			Expect(err).ToNot(HaveOccurred())
 			Expect(finished).To(BeFalse())
 		})
 	})
-	When("InProgress status is True", func() {
+	When("Clean up annotation should be in progress", func() {
```

Comment:
you are asserting on the `finished` var instead of `annotation`, thus could you change the description to `When clean up annotation is Done \ In Progress` etc?